recipes/seqlogo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "seqLogo" %}
+{% set version = "0.0.6" %}
+{% set sha256 = "606d6e963debe4ed5dc8b2d490e00de87f05a577f07e01329398bc9028e7f9d1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True # [win32]
+  skip: True # [win64]
+  skip: True # [py<34]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - numpy
+    - pandas
+    - pip
+    - python
```

Comment:
put the `<3.4` here

Are you sure that `<3.4` is what you mean? It's rare to support 3 but not 3.5+.

Reply:
d403c33fb3dac1231cde6a792a425597ae1eff62 resolves this